### PR TITLE
Use `GetDigestOIDForSignatureAlgorithm` to set the digest algorithm OID

### PR DESCRIPTION
**Pull request by malancas**
Small follow up PR to https://github.com/digitorus/pkcs7/pull/11. This pull request uses `pkcs7.GetDigestOIDForSignatureAlgorithm` to set the digest algorithm used in the signing process. Currently the digest algorithm is hardcoded to SHA-256. This PR also fixes a few small typos.

### CODE REVIEW

**Comment by vanbroup** [comment]
Can you check the lint errors and test coverage?

**Comment by malancas** [comment]
Sure, I'll take a look

**Comment by malancas** [comment]
Reviewing the linter failure, it looks like it could be due to the linter GitHub Action being out of date. Updating to `golangci/golangci-lint-action@v3` may help.

Reviewing the codecov failures, it looks like most of them are responding to code changes that fix typos. Do you know if the codecov warnings can be disabled for these lines?

**Comment by malancas** [comment]
I updated the golangci-lint Action to the latest version, I think an older version was causing the failure. I also pushed a codecov config that disables alerts for patching since the alerts are noisy but we can make an adjustments to this file. I added this because most of the patch alerts in this PR are around typo fixes.

**Comment by malancas** [comment]
@vanbroup just wanted to follow up around the linter and codecov updates as resolution to the workflow failures.

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+coverage:
```
Comment (vanbroup) [inline codecov.yml:1]: I don't think it's the intention to turn off the coverage for the project. In general coverage should almost never go down, we also had no previous issues with this.

**Pull request merged**